content/departments/product-engineering/engineering/cloud/iam-and-admin-exp/index.md

```diff
@@ -27,14 +27,18 @@ The Growth & Integration team also works on the [new user experience](../../../p
 
 ### Product domains
 
+#### Identity and access management
```

Comment:
Basically, I moved these first two items up as they are our main areas of focus and edited/removed what was Cloud related